src/runtime_src/core/tools/common/OptionOptions.cpp

```diff
@@ -20,8 +20,8 @@
 #include <iostream>

```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
comment addressed.